nuxeo-webdav/src/main/java/org/nuxeo/ecm/webdav/resource/ExistingResource.java

```diff
@@ -172,13 +214,16 @@ private Response copyOrMove(String method, @HeaderParam("Destination") String de
         destPath = destinationBackend.parseLocation(destPath).toString();
         log.debug("to " + davDestPath);
 
-        // Remove dest if it exists and the Overwrite header is set to "T".
+        boolean isOverwrite = "T".equals(overwrite);
```

Comment:
Why removing the comment ? Also, comment references `"T"` but we used to check `"F"`. Are you sure about checking `"T"` ? I would go to update comment instead.

Reply:
Because the document is no longer removed.